plugins/woocommerce-admin/client/marketplace/components/woocom-connect-notice/woocom-connect-notice.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { connectUrl } from '~/marketplace/utils/functions';
+import Notice from '~/marketplace/components/notice/notice';
+import { getAdminSetting } from '~/utils/admin-settings';
+
+export default function WoocomConnectNotice(): JSX.Element | null {
```

Comment:
I think `ConnectNotice` as a component name looks fine. Other marketplace component names aren't prefixed with `Woocom` as the path when importing already indicate it.

Reply:
this suggestion is okay, I will update it